### PR TITLE
Update NetAppFiles to 2022-05-01

### PR DESCRIPTION
**Pull request by hildurhodd**
# Description
This is a copy of https://github.com/Azure/azure-sdk-for-python/pull/26260/ with new recordings

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/netapp/azure-mgmt-netapp/tests/test_snapshot_policies.py [hunk 1]
```diff
@@ -54,10 +53,11 @@ class TestNetAppSnapshotPolicy(AzureMgmtRecordedTestCase):
     def setup_method(self, method):
         self.client = self.create_mgmt_client(azure.mgmt.netapp.NetAppManagementClient)
         if self.is_live:
-            self.network_client.virtual_networks.begin_delete(TEST_RG, VNETNAME)
+            from azure.mgmt.network import NetworkManagementClient
+            self.network_client = self.create_mgmt_client(NetworkManagementClient)
 
 
-    # Before tests are run live a resource group needs to be created along with vnet and subnet
+            # Before tests are run live a resource group needs to be created along with vnet and subnet
```
Comment (msyyc) [inline sdk/netapp/azure-mgmt-netapp/tests/test_snapshot_policies.py:60]: @hildurhodd Why do you add the blank space

**Comment by msyyc** [comment]
@hildurhodd Why did you change the configuration? Please keep it same with main branch
image
image

**Comment by hildurhodd** [comment]
> @hildurhodd Why did you change the configuration? Please keep it same with main branch image image

I noticed after I created the pull request that this was different than in the original PR so I reverted it. It should be the same https://github.com/Azure/azure-sdk-for-python/pull/26260/files

File: sdk/netapp/azure-mgmt-netapp/tests/test_snapshot_policies.py (hunk 1 above)
Comment (hildurhodd) [reply sdk/netapp/azure-mgmt-netapp/tests/test_snapshot_policies.py]: This was of course a mistake; I have already fixed it.

**Comment by hildurhodd** [comment]
@msyyc I have removed the blank line and the autorest version in the same as main. Can we get this moving?

**Review by msyyc — APPROVED** [review]

**Comment by msyyc** [comment]
https://github.com/Azure/sdk-release-request/issues/3179

**Pull request merged**